docs/content/en/docs/fleet-manager/distributedstorage-plugin.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Enable Policy Management with fleet"
+linkTitle: "Policy Management"
+weight: 30
+description: >
+  The easiest way to enable policy management with fleet.
+---
+
+In this tutorial, we will cover how to implement unified distributed storage on a set of clusters, using [Fleet](https://kurator.dev/docs/references/fleet-api/#fleet)
+
+## Architecture
+
+Fleet's unified distributed storage is built on top of [Rook](https://rook.io/), and the overall architecture is shown below:
+
+{{< image width="100%"
+    link="./image/fleet-policy.drawio.svg"
+    >}}
```

Comment:
this's wrong.

Reply:
good catch


@LiZhenCheng9527 You can read the preview link